tests/abstract5-test.toit

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2021 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import expect show *
+
+class A:
+  foo: return 1
+
+abstract class B extends A:
+  abstract foo x=null
+
+class C extends B:
+  foo x: return 2
+
+abstract class D:
+  abstract foo x=null
+  foo: return 1
+
+class E extends D:
+  foo x: return 2
+
+main:
+  c := C
+  expect_equals 1 c.foo
```

Comment:
```suggestion
  expect-equals 1 c.foo
```

Reply:
I will migrate all code in a follow-up PR.